explorer/interpreter/value.cpp

```diff
@@ -33,19 +35,63 @@ auto StructValue::FindField(std::string_view name) const
   return std::nullopt;
 }
 
-static auto GetMember(Nonnull<Arena*> arena, Nonnull<const Value*> v,
-                      const FieldPath::Component& field,
-                      SourceLocation source_loc, Nonnull<const Value*> me_value)
+static auto GetSpecialMember(Nonnull<const Value*> v,
```

Comment:
"Special member" isn't very clear. I'd recommend calling this `GetBaseClassMember`

Reply:
This was meant to be generic (nominal, positional, and ... "the rest"). But we can still rename it as today we don't have any other. Would that be preferred?

Reply:
Made changes in that direction